timealign/settings.py

```diff
@@ -25,7 +25,12 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
+
 ALLOWED_HOSTS = []
+if DEBUG:
+    from subprocess import Popen, PIPE
+    hostname = Popen(('hostname'), stdout=PIPE).communicate()[0].decode().strip()
+    ALLOWED_HOSTS.append(hostname)
```

Comment:
My local server runs on 127.0.0.1 rather than `hostname`, would you be so kind as to add that :-)